bucket_manifest/utils.py

```diff
@@ -0,0 +1,86 @@
+import csv
+import random
+from datetime import datetime
+import string
+import logging
+import boto3
+from botocore.exceptions import ClientError
+
+
+def randomString(stringLength=10):
+    """Generate a random string of fixed length """
+
+    letters = string.ascii_lowercase
+    return "".join(random.choice(letters) for i in range(stringLength))
+
+
+def write_csv(filename, files, fieldnames=None):
+    """
+    write to csv file
+    Args:
+        filename(str): file name
+        files(list(dict)): list of file info
+        [
+            {
+                "GUID": "guid_example",
+                "filename": "example",
+                "size": 100,
+                "acl": "['open']",
+                "md5": "md5_hash",
+            },
+        ]
+        fieldnames(list(str)): list of column names
+    Returns:
+        filename(str): file name
+    """
+
+    if not files:
+        return None
+    fieldnames = fieldnames or files[0].keys()
+    with open(filename, mode="w") as outfile:
+        writer = csv.DictWriter(outfile, delimiter="\t", fieldnames=fieldnames)
+        writer.writeheader()
+
+        for f in files:
+            writer.writerow(f)
+
+    return filename
+
+
+def upload_file(
+    file_name,
+    bucket,
+    object_name=None,
+    aws_access_key_id=None,
+    aws_secret_access_key=None,
+):
+    """Upload a file to an S3 bucket
+    :param file_name: File to upload
+    :param bucket: Bucket to upload to
```

Comment:
using different docstring format here, we should stick with one style for consistency. I prefer the one above using "Args:" and whatnot